Adjust docs to account for SHConfig changes (#443)
sentinel-hub/sentinelhub-py: README.md

## Installation

> The package requires a Python version >= 3.8. The package is available at the PyPI package index and can be installed with

Reviewer: copied from the install section below

```suggestion
This package requires Python >=3.8 and can be installed with the PyPI package manager:
```
